.gitignore

```diff
@@ -8,3 +8,4 @@ wp-phptidy.php
 /account_cleanup/vendor/
 /vendor/
 /rules/
+composer.lock
```

Comment:
It can go either way but, in thinking about it, we test this thoroughly with a specific combination and this is closer to an application than a library ... probably best to keep. Will add back!